datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/service_checks.py

```diff
@@ -31,6 +54,7 @@ def service_checks():
         manifest = load_manifest(check_name)
         service_check_relative = manifest.get('assets', {}).get('service_checks', '')
         service_checks_file = os.path.join(root, check_name, *service_check_relative.split('/'))
+        manifest_file = os.path.join(root, check_name, 'manifest.json')
```

Comment:
https://github.com/DataDog/integrations-core/blob/663c746b692286bff1a86886074451731f4204bb/datadog_checks_dev/datadog_checks/dev/tooling/utils.py#L173-L180

Reply:
Actually, the manifest was already loaded 3 lines above ...